variables.tf

```diff
@@ -497,3 +497,81 @@ variable "log_analytics_solution_promotion_code" {
   description = "A promotion code to be used with the solution"
   default     = ""
 }
+
+# Networking
+variable "resource_group_name" {
+  type    = string
+  default = null
+  description = "Name of pre-exising resource group. Leave blank to have one created"
+}
+
+variable "vnet_name" {
+  type    = string
+  default = null
+  description = "Name of pre-exising vnet. Leave blank to have one created"
+}
+
+variable "vnet_address_space" {
+  type        = string
+  default     = "192.168.0.0/16"
```

Comment:
Just to clarify this value replaces the old `10.0.0.0/16` CIDR range from the azure_vnet variables file.

Reply:
yes